File: src/main/java/MoveZeroes/MoveZero.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        nonZero_count = 0

        for i in range(0, len(nums)):
            if nums[i] is not 0:
                nums[nonZero_count] = nums[i]
                nonZero_count += 1

        for i in range(nonZero_count, len(nums)):
            nums[i] = 0
        return nums

File: src/main/java/MoveZeroes/AzusaMutan/MoveZero.java
public class MoveZeroes {
    public static void moveZeroes(int[] nums) {
        int m;
        for(int i = 0 ; i < nums.length; i++) {
            if (nums[i] == 0) {
                m = i;
                for (int j = i; j < nums.length; j++) {
                    if (nums[j] != 0) {
                        int temp = nums[m];
                        nums[m] = nums[j];
                        nums[j] = temp;
                        m = j;
                    }
                }
            }
        }
    }
}
